docs/TempUG.md

```diff
@@ -330,78 +332,98 @@ Note you must provide a 'name' not already in the address book.
 
 <box type="tip" seamless>
 
 **Tips:**
-- Tip 1
-- Tip 2
+If the user is currently in ___SCHEDULE list___, the command can be shortened to `add`.
 
 </box>
 
 Example usages:
-* `some code here`
-* `another code here`
+* `addLesson -name Chemistry Lesson at Bishan`
+* `addLesson -name Lesson at Tai Seng -start 09:00 -end 11:00 -day 03/21 -subject physics`
+* In ___SCHEDULE list___ :
+  * `add -name Lesson at Yishun -day 21 -subject MATHEMATICS`
+
 
 Success outputs:
-* Input: `code with compulsory parameters`
-* Input: `code with compulsory and optional parameters`
+* Input: `addLesson -name Chemistry Lesson at Bishan`
 ```
-This block of code is for success outputs
+New lesson added: Lesson Chemistry Lesson at Bishan
+```
+* Input: `addLesson -name Lesson at Tai Seng -start 09:00 -end 11:00 -day 03/21 -subject physics`
+```
+New lesson added: Lesson Lesson at Tai Seng from 9:00 AM to 11:00 AM on 21-03-2023 for PHYSICS
 ```
+
+
 Failure outputs:
-* Input: `invalid command code here`
-  * Error: Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+* Input: `addLesson -name Chemistry Lesson` followed by `addLesson -name Chemistry Lesson -day 2023/12/12`
+  * Error: A new lesson cannot have the same name as an existing lesson. Enter another name for the new lesson.
 ```
-Invalid command with the error message here
+Lesson with this name already exists in the schedule
 ```
 
-* Input: `invalid command code here`
-  * Error: Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+
+* Input: `addLesson -name English Lesson -start 13:00 -end 10:00`
+  * Error: The start time specified for a lesson cannot be after its end time. Enter a start time that is before the end time.
 ```
-Invalid command with the error message here
+Invalid lesson input: End time: 10:00 AM cannot be before start time: 1:00 PM.. 
+Usage: addLesson -name NAME (any number of unique [-subject|day|start|end VALUE]). 
+ For example, addLesson -name John -subject English -day 23 -start 14:30 -end 16:30
+ If you are currently displaying schedule list, you could use 'add' inplace of 'addLesson'. 
+ Note you must provide a 'name' not already in the schedule and 'start' must be before 'end'.
 ```
+![Failure for addLesson](images/add-lesson/add_lesson_failure2.png)
+
 
 #### For task list:
 
-Format: `command COMPULSORY [optional]` (for list specific format)
-* Format info 1
-* Format info 2
+Format: `addTask [INDEX] DESCRIPTION` 
+* Adds the task to the lesson at specified `INDEX` of the displayed ___SCHEDULE list___.
+* A new task cannot have the same description as existing tasks in the task list of that specific lesson.
+* Refer to the parameter constraints [here](#parameter-summary).
+
 
 <box type="tip" seamless>
 
 **Tips:**
-- Tip 1
-- Tip 2
+The command can be shortened to `addTask DESCRIPTION` if the task is to be added to the shown lesson.
```

Comment:
Emm, since we put "[INDEX]" as an optional field here, probably we should explain in what case we can have index as optional right after the format

Reply:
Make the necessary changes in the recent commit, thank you. Can help verify, not sure whether this is what you meant